src/main/java/com/gd/clinic/entities/Event.java

```diff
@@ -0,0 +1,39 @@
+package com.gd.clinic.entities;
+
+import lombok.*;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "Events")
```

Comment:
please call table from lowercase

Reply:
Corrected that this commit, will use lowercase from now on